test/Libraries/Microsoft.Extensions.AI.AotCompatibility.TestApp/Microsoft.Extensions.AI.AotCompatibility.TestApp.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net9.0</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFramework>net9.0</TargetFramework>
```

Reply:
I had that initially, but changed it to TargetFrameworks because using TargetFramework fails:
```
Invalid restore input. Duplicate frameworks found: 'net9.0, net9.0, net9.0'.
```
Presumably it's something about how multitargeting is arranged repo-wide, and I didn't bother to dig deeper.

Reply:
It should be `TargetFrameworks` because the multi-targeting and tfm-overrides.